pkg/perf/cache.go

```diff
@@ -0,0 +1,49 @@
+package perf
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// TestResultData the result test schema
+type TestResultData struct {
+	TestName   string
+	TestNumber uint64
+	Result     interface{}
+}
+
+// GetRequest the get request struct
+type GetRequest struct {
```

Comment:
this struct can be removed completely and use the task name always to get the task result from the cache